elliottlib/cli/common.py

```diff
@@ -94,3 +93,15 @@ def find_default_advisory(runtime, default_advisory_type, quiet=False):
         ', '.join(constants.standard_advisory_types)))
 
 pass_runtime = click.make_pass_decorator(Runtime)
+
+
+def coro(f):
```

Comment:
Could you name this `coroutine` or `coroutine_wrapper`? It conveys the meaning more readily (I needed to browse here to find out what `coro` would be about).

Reply:
I actually copied it from the linked GitHub issue without any modifications. Maybe I should rename it to `click_ coroutine`?